INSTALL.md

```diff
@@ -1,26 +1,27 @@
----
 ## Mysterium VPN node (Any OS with Docker)
 
-Most convenient way requiring least configuration is to run a node using [Docker](https://docs.docker.com/install/). 
-All node versions are available through docker hub: 
- 
+Most convenient way requiring least configuration is to run a node using [Docker](https://docs.docker.com/install/).
+All node versions are available through docker hub:
+
 https://hub.docker.com/r/mysteriumnetwork/myst/
 
 ### Fetching and running docker image
 ```bash
-sudo docker run --cap-add NET_ADMIN --net host --name mysterium-node -d mysteriumnetwork/mysterium-node --agreed-terms-and-conditions
+sudo docker run --cap-add NET_ADMIN --net host --name myst -d mysteriumnetwork/myst service --agreed-terms-and-conditions
 ```
 
-Note: to run server, you will have to accept terms & conditions by adding '--agreed-terms-and-conditions' command line option.
-Note 2: it's mandatory to run docker container with --net host to correctly detect VPN service ip which needs to be published to clients, assuming that host on which node is running has external interface with public ip
+>**Note:** to run server, you will have to accept terms & conditions by adding '--agreed-terms-and-conditions' command line option.
```

Comment:
We sync Node WIKI with same running guide. Update it also ;)

Reply:
I was planning to do it when this PR will be merged after review.